observability/grafana-dashboards/kas-monitor/kas_monitor_api_slo.json

```diff
@@ -844,16 +844,13 @@
     "list": [
       {
         "allowCustomValue": false,
-        "current": {
-          "text": "Managed_Prometheus_hcps-usw3scla",
-          "value": "hcps-usw3scla"
-        },
+        "current": {},
         "includeAll": true,
         "name": "datasource",
         "options": [],
         "query": "prometheus",
         "refresh": 1,
-        "regex": "",
+        "regex": "^Managed_Prometheus_hcps-.*$",
```

Comment:
What other ones could possibly exist? If we have leftover crud from previous attempts, I would rather see us remove them than hide them.